modules/nextflow/src/main/groovy/nextflow/fusion/FusionEnvProvider.groovy

```diff
@@ -46,4 +49,32 @@ class FusionEnvProvider {
             result.FUSION_CACHE_SIZE = "${config.cacheSize().toMega()}M"
         return result
     }
+
+    protected Map<String,String> getFusionEnvironment(String scheme, FusionConfig config) {
+        final list = Plugins.getExtensions(FusionEnv)
+        log.debug "Fusion environment extensions=$list"
+        final result = new HashMap<String,String>()
+        for( FusionEnv it : list ) {
+            final env = it.getEnvironment(scheme,config)
+            if( env )
+                result.putAll(env)
+        }
+        return result
+    }
+
+    protected Map<String,String> getFusionToken(String scheme, FusionConfig config) {
+        final providers = Plugins.getPriorityExtensions(FusionToken)
+        log.debug "Fusion token extensions=$providers"
+        final result = new HashMap<String,String>()
+        try {
+            final env = providers.first.getEnvironment(scheme,config)
```

Comment:
Don't know how this got through, but this is causing CI errors. Should be:
```groovy
            final env = providers.first().getEnvironment(scheme,config)
```

Reply:
Uhh tricky. `first()` is a Groovy extension method, `getFirst()` is Java 21 default method

Reply:
Done a0ca6e4f